src/components/Home.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+import LessonCard from './LessonCard';
+import EmailBar from './EmailBar';
+
+class Home extends Component {
+  render() {
+    return (
+        <div className="home" id="main">
+          <div className="preview">
+            <h1>Preview</h1>
+            <div className="lesson-container">
+              <LessonCard />
```

Comment:
Are these `<LessonCard />` always fixed, or should they be populated from a list? [even for placeholder purposes]